### PR TITLE
Added Advice Section in Treatment Summary

### PR DESCRIPTION
**Pull request by peeyush14goyal**
Advice is shown from `last_consultation` field

In case of data its shown like
image

If there is no data then 
image

### CODE REVIEW

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/AssetCreate.tsx [hunk 1]
```diff
@@ -160,6 +160,10 @@ const AssetCreate = (props: AssetProps) => {
         serial_number: serial_number,
         warranty_details: warranty_details,
         location: location,
+        vendor_name: "XYZ",
```
Comment (bodhish) [inline src/Components/Facility/AssetCreate.tsx]: Unrelated change

**Review by bodhish — APPROVED** [review]

File: src/Components/Facility/AssetCreate.tsx (hunk 1 above)
Comment (peeyush14goyal) [reply src/Components/Facility/AssetCreate.tsx]: @bodhish I was testing some other issue. 😅. Removed

**Comment by nihal467** [comment]
test approved
